src/signer/ammv2.ts

```diff
@@ -0,0 +1,33 @@
+// sign order from custom quoter no need to put it in MMSK
+import { assetDataUtils, generatePseudoRandomSalt } from '0x-v2-order-utils'
+import * as cryptoRandomString from 'crypto-random-string'
+import { orderBNToString } from '../utils'
+import { NULL_ADDRESS } from '../constants'
+
+export const buildSignedOrder = (order, payload, makerAddress, wethAddress): any => {
+  const makerAssetAddress = order.makerAssetAddress.toLowerCase()
+  const takerAssetAddress = order.takerAssetAddress.toLowerCase()
+  // = Rewrite order fields
+  // 1. change maker address to LP pool address
+  order.makerAddress = makerAddress
+  // 2. convert weth to eth
+  if (makerAssetAddress === wethAddress.toLowerCase()) {
+    order.makerAssetData = assetDataUtils.encodeERC20AssetData(NULL_ADDRESS)
+  } else {
```

Comment:
Should overwrite makerAssetAddress and takerAssetAddress here.
Already fixed the bug for AMMV1, please find https://github.com/consenlabs/tokenlon-mmsk/pull/23/commits/995f158b2c724335f88ab422a83272e0d067d152#diff-53fe56f13488d83999f9e39bbbeaf0faab6702e3aa4176aa6215ea679935a27aR15

Reply:
Fixed

Reply:
Fixed